src/syntax/parser.rs

```diff
@@ -376,7 +377,7 @@ impl<TokType: Tokenizer + Iterator<Item=Tokens>> Parser<TokType> {
 
         let expr = Expr::FnDecl(fn_name, args, return_type, definition);
 
-        return Some(ExprWrapper::default(expr));
+        Some(ExprWrapper::default(expr))
```

Comment:
bit me on my below comment a little. The difference between `some_long_function_call(or_even_maybe_it_wasn't_that_long)` and `some_long_function_call(or_even_maybe_it_wasn't_that_long);` is 1 character, and inside a match or long function, it just obfuscates where the end of the function is. Also if you're in a big match (like below), then you can and do end up with something like 
```
let result = match thing {
    ASDF => None,
    ...50 lines later....
}
```
vs 
```
match thing {
    ASDF => None,
    ...50 lines later....
}
```
where the first just assigns to result, and the second returns the function, so you need to look at the beginning of the scope to know if you're assigning the result to the variable or the function return value